Tasks/VsTest/vstest.ts

```diff
@@ -874,9 +934,17 @@ function responseContainsNoTests(filePath: string): Q.Promise<boolean> {
     return utils.Helper.readFileContents(filePath, 'utf-8').then(function (resp) {
         if (resp === '/Tests:"' || resp === '/Tests:' || resp === '/TestCaseFilter:') {
             return true;
-        }
-        else {
+        } else {
             return false;
         }
     });
+}
+
+function publishEventToCi(areaCode : string, message: string, tracepoint: string, isusererror: string = 'false') {
```

Comment:
can this be a int (inline with service side)

Reply:
Had started with ints. Nitin had initially asked that i make it a string for some reason. Will revert to integers starting from 1001.

---
In reply to: [136512464](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136512464) [](ancestors = 136512464)